perses/rjmc/topology_proposal.py

```diff
@@ -1115,18 +1116,18 @@ def _construct_atom_map(self,
                             residue_map,
                             old_topology,
                             new_topology,
-                            extra_sidechain_map=None,
-                            demap_CBs=False):
+                            extra_sidechain_map=None):
         """
         Construct atom map (key: index to atom in new residue, value: index to atom in old residue) to supply as an argument to the TopologyProposal.
         
         By default, the atom map:
         1) Maps all backbone atoms (exception: for GLY, do not map HA2 and HA3)
-        2) Map CBs only (no other sidechain atoms)
-        
+        2) Map CBs, HBs, and CGs.
+        3) Also maps HG2s, HG3s, and CDs, but not CD1, CD2, HG, HG1, HG11, HG12, HG13, HG21, HG22, HG23 (as mapping the latter atoms may match dissimilar geometries)
```

Comment:
this appears different from the logic that i thought we agreed upon? im not sure

Reply:
How is it different than the logic we agreed upon?

Reply:
I actually don't think we talked about this during our chat this week. Here, I am mapping up to CG atoms for most transformations. However, I also map CD and HG2 and HG3 if there are direct atom matches. There are only a few transformations where this is true -- transformations where the old and new residue are both in the set of {LYS, ARG, GLU, GLN}. I don't want to try to search for non-exact atom matches involving CD and HG atoms because there is a risk for mapping differing sidechain geometries to each other, e.g. I don't want to accidentally map CD from LYS onto the CD1 in TYR.